issue #95 - add salt to md5 and email anonymizers (#96)
makinacorpus/DbToolsBundle: docs/content/anonymization/core-anonymizers.md

```
:::

:::tip
Username value is salted prior to be hashed using md5 in order to prevent reverse hashing
with rainbow tables, but salt is global across the same anonymizer run, which keeps a single

> value when the same email is anonymized more than once.

Author: Actually I changed "username" to "email" (will push that soon) it's more correct.

```
:::

> :::tip

Author: As you wish, both are fine because it also document how to deactivate, but let's go for info.

> Hashing a string is not anonymizing it because hash functions have a reproducible output,
> this in order to avoid decrypting data using rainbow tables, per default a salt will be
> added to string values prior to hasing.

Reviewer: ```suggestion
Hashing a string is not anonymizing it because hash functions have a reproducible output.
In order to avoid decrypting data using rainbow tables, a salt will be added by default to
string values prior to hashing.
```
